libs/api/domains/cms/src/lib/cms.resolver.ts

```diff
@@ -368,7 +368,7 @@ export class CmsResolver {
     @Args('input') { lang, slug }: GetSingleNewsInput,
   ): Promise<News | null> {
     return this.cmsElasticsearchService.getSingleDocumentTypeBySlug<News>(
-      getElasticsearchIndex(lang),
+      getElasticsearchIndex(lang ?? 'is'),
```

Comment:
It looks like these fields have default values in dto but are marked as optional in typescript.
We should be able to remove the optional mark in TS since they should always be set, that would negate the need for checks here

Reply:
Good point